c2cgeoportal/views/entry.py

```diff
@@ -1276,11 +1277,12 @@ def logout(self):
         return set_common_headers(
             self.request, "logout", NO_CACHE,
             response=Response("true", headers=headers),
+            add_cors=True
         )
 
     @view_config(route_name="loginchange", renderer="json")
     def loginchange(self):
-        set_common_headers(self.request, "loginchange", NO_CACHE)
+        set_common_headers(self.request, "loginchange", NO_CACHE, add_cors=True)
```

Comment:
I don't understand the relation between CORS and CSRF to change the password eventually about reading the result but not change the password.
For me we already have an issue...
And if we don't do that we cant do an ngeo example to demo server ... (for the concerned functionalities)
I am wrong?